projects/Mallard/src/screens/issue-screen.tsx

```diff
@@ -151,6 +151,13 @@ const IssueFronts = ({
 }) => {
     const { container, card } = useIssueScreenSize()
     const { width } = useDimensions()
+    const ref = useRef<FlatList<any> | null>()
+
+    useEffect(() => {
+        if (ref && ref.current && ref.current.scrollToOffset) {
```

Comment:
So close to 3.7 where we would be able to write `ref?.current?.scrollToOffset?.({ animated: false, offset: 0 })` ... will update TS soon but until then this looks good!